ui-common/src/components/modals/TrafficStatsModal/TimelineBarChart/TimelineBarChart.module.sass

```diff
@@ -0,0 +1,5 @@
+.barChartContainer
+    width: 100%
+    display: flex 
+    justify-content: center
+    
```

Comment:
new line

Reply:
Its suppose to have one line gap isn't it?